### PR TITLE
Adding support for django-nested-admin

### PR DESCRIPTION
**Pull request by trumpet2012**
Added support for django-nested-admin by moving the initialization function to a separate media script that watches for the `data-django-jsonform` data attribute and uses its json value to configure the form. This allows django-nested-admin to update the input IDs while allowing django-jsonform to still be able to find and initialize itself correctly.

(The other PR had a bad branch name)

### CODE REVIEW

**Review by bhch — CHANGES_REQUESTED** [review]
Hello, thank you for this Pull Request. Really appreciate the work you've done here.

I've left a few comments with requests for changes. Everything else looks good.

File: django_jsonform/static/django_jsonform/index.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
+(function() {
+  window.addEventListener("load", function() {
```
[the remark this answers is not in the document]
Comment (trumpet2012) [reply django_jsonform/static/django_jsonform/index.js]: Good point, I've updated it to use `DOMContentLoaded`.

File: django_jsonform/widgets.py [hunk 2]
```diff
@@ -46,21 +46,27 @@ def render(self, name, value, attrs=None, renderer=None):
...
-            'model_name': self.model_name,
-            'data': value or json.dumps(''),
-            'schema': json.dumps(schema),
-            'file_handler': self.file_handler or get_setting('FILE_HANDLER', ''),
-            'error_map': getattr(self, 'error_map', {}),
-            'validate_on_submit': self.validate_on_submit
+            'config': {
+                'fieldName': context['widget']['name'],
+                'modelName': self.model_name,
+                'data': value or '',
```
[the remark this answers is not in the document]
Comment (trumpet2012) [reply django_jsonform/widgets.py]: Ah yeah that makes sense. I've update that as well.

File: django_jsonform/widgets.py [hunk 3]
```diff
@@ -78,6 +84,7 @@ def media(self):
             'django_jsonform/vendor/react-dom.production.min.js',
             'django_jsonform/vendor/react-modal.min.js',
             'django_jsonform/react-json-form.js',
+            "django_jsonform/index.js",
```
[the remark this answers is not in the document]
Comment (trumpet2012) [reply django_jsonform/widgets.py]: fixed.

**Comment by trumpet2012** [comment]
@bhch thanks for the feedback and catching those issues! It should be good for you to re-review again.

**Comment by bhch** [comment]
Hello, I've released this in [v2.13.0](https://github.com/bhch/django-jsonform/releases/tag/v2.13.0). Thank you for your contribution.

**Pull request merged**